### ISSUE TITLE
No updates in forever

### ISSUE THREAD

**Issue by klikevil** [minipli/linux-unofficial_grsec#31]
Can we get some more commits?  I'd like to be on a more modern kernel as I have some newer hardware in my server.

**Comment by klikevil** [comment]
I guess this has been abandoned :-/

**Comment by minipli** [comment]
Yeah, unfortunately I don't have the time any more to maintain this fork -- neither would I be allowed to, as my currently employment would be in conflict with continuing this work. :(

**Comment by klikevil** [comment]
Ah oh well, thanks so much for the releases you did do!  I wish spengler 
would sell individual consumer licenses :-/
